### DISCUSSION TITLE
Accessing discussions

### DISCUSSION THREAD

**Discussion by comsma** [weather-gov/api#630] [General]
I would like to access discussions such as [this one](https://forecast.weather.gov/product.php?site=LOT&issuedby=LOT&product=AFD&format=CI&version=1&glossary=1) via a get request. I didn't see a whole lot of documentation regarding accessing the discussions. Where can I pull this data?

**Comment by sullynole** [comment]
Text products can be found under the products endpoint.

https://api.weather.gov/products?type=AFD&office=KLOT

Check Specifications tab here for more details: https://www.weather.gov/documentation/services-web-api
